bpf/bpf_host.c

```diff
@@ -902,12 +902,20 @@ handle_netdev(struct __ctx_buff *ctx, const bool from_host)
 	return do_netdev(ctx, proto, from_host);
 }
 
+/*
+ * from-netdev is attached as a tc egress filter
+ * to the node's 'cilium_net' interface.
+ */
 __section("from-netdev")
 int from_netdev(struct __ctx_buff *ctx)
 {
 	return handle_netdev(ctx, false);
 }
 
+/*
+ * from-host is attached as a tc egress filter
+ * to the node's 'cilium_host' interface.
```

Comment:
In case of Flannel chaining, `{to,from}-host` can also be attached to the flannel master device. See https://github.com/cilium/cilium/blob/6083b066e76ae07e0c1fad0d18ca038d578ef533/pkg/endpoint/endpoint.go#L462-L465

Reply:
Thanks, added!